brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapViewController.swift

```diff
@@ -263,12 +263,6 @@ class SwapViewController: BaseTableViewController<SwapCoordinator,
                     
                     self?.interactor?.confirm(viewAction: .init())
                 } else {
-                    self?.coordinator?
```

Comment:
we need this. sandra approved that it is ok to have it only on coordinator supported flows @rcresnik